lib/options.go

```diff
@@ -285,6 +285,8 @@ type Options struct {
 
 	// Redirect console logging to a file
 	ConsoleOutput null.String `json:"-" envconfig:"K6_CONSOLE_OUTPUT"`
+
+	Nic null.String `json:"nic,omitempty" envconfig:"K6_NIC"`
```

Comment:
As I mentioned in the CLI flag configuration, this should probably be a string slice.